servicetest/filesystem/test_filesystem.py

```diff
@@ -41,7 +46,7 @@ def logfile_path():
     Container.READONLY: False
 }
 
-@pytest.mark.usefixtures("dbus_launch", "agent", "assert_no_proxy")
+@pytest.mark.usefixtures("create_testoutput_dir", "dbus_launch", "agent", "assert_no_proxy")
 class TestFileSystem(object):
     """ This suite should do some basic testing of the Filesystem within the
         containers and that the whole chain is working properly.
```

Comment:
I think it would be nice if all test suites followed the same pattern w.r.t where the files go and what's in the .gitignore file, so change it if it can be done without any major hassle IMO.